packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -87,6 +88,10 @@ class ReorderableListView extends StatefulWidget {
   /// List [children] can only drag along this [Axis].
   final Axis scrollDirection;
 
+  /// Creates a ScrollPosition to manage and determine which portion
+  /// of the content is visible in a scroll view.
```

Comment:
I would further elaborate here on how it is used for `ReorderableListView`:
```dart
/// Creates a [ScrollPosition] to manage and determine which portion
/// of the content is visible in the scroll view.
///
/// This can be used in many ways, such as setting an initial scroll offset, 
/// (via [ScrollController.initialScrollOffset]), reading the current scroll position 
/// (via [ScrollController.offset]), or changing it (via [ScrollController.jumpTo] or 
/// [ScrollController.animateTo]).
```